js/modules/k6/experimental/tracing/client.go

```diff
@@ -231,6 +194,61 @@ func (c *Client) Put(url goja.Value, args ...goja.Value) (*httpmodule.Response,
 	return c.Request(http.MethodPut, url, args...)
 }
 
+func (c *Client) instrumentedCall(call func(args ...goja.Value) error, args ...goja.Value) error {
+	if len(args) == 0 {
+		args = []goja.Value{goja.Null()}
+	}
+
+	traceContextHeader, encodedTraceID, err := c.generateTraceContext()
+	if err != nil {
+		return err
```

Comment:
nit: this Pr would've been easier to code review if you didn't move this methods around in the codebase

Reply:
Fair enough 🙇‍♂️Sorry for that. I'm trying to get better at isolating these kind of changes, but still having a hard time.